Spectogram.py

```diff
@@ -4,16 +4,26 @@
 import scipy.signal
 import matplotlib.pyplot as plt
 from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
+ gui-performance-enhancement
+import threading
+import queue
 import subprocess
 
 
+main

```

Comment:
could you remove branch names mentioned across the file for "main" and "gui-performance-enhancement"

Reply:
Means you are saying that I have to remove the branch name
Is it correct I am saying

Reply:
yes, remove the branch names in the code file. I believe it might've occurred while solving a merge conflict

Reply:
Ok I will do thanks for the info

Reply:
Can you please check it now for merging

Reply:
> could you remove branch names mentioned across the file for "main" and "gui-performance-enhancement"

I still see those!